### PR TITLE
Update/add flake bot labels

### PR DESCRIPTION
**Pull request by keyonghan**
fixes: https://github.com/flutter/flutter/issues/130995

### CODE REVIEW

File: app_dart/lib/src/request_handlers/flaky_handler_utils.dart [hunk 1]
```diff
@@ -417,6 +422,16 @@ String? getTeamLabelFromTeam(Team? team) {
       return kToolLabel;
     case Team.web:
       return kWebLabel;
+    case Team.infra:
```
Comment (zanderso) [inline app_dart/lib/src/request_handlers/flaky_handler_utils.dart:425]: Can you use pattern matching here?

```dart
return switch (team) {
  Team.framework => kFrameworkLabel,
  Team.engine => kEngineLabel,
  ...
  null => null,
};
```

Comment (keyonghan) [reply app_dart/lib/src/request_handlers/flaky_handler_utils.dart]: Updated.

File: app_dart/lib/src/request_handlers/flaky_handler_utils.dart [hunk 2]
```diff
@@ -408,19 +413,19 @@ String _issueBuilderLink(String? builder) {
...
+    Team.engine => kEngineLabel,
+    Team.tool => kToolLabel,
+    Team.web => kWebLabel,
+    Team.infra => kInfraLabel,
+    Team.android => kAndroidLabel,
+    Team.ios => kIosLabel,
+    Team.release => kReleaseLabel,
+    Team.plugins => kEcosystemLabel,
+    Team.unknown => null,
+    null => null,
```
Comment (zanderso) [inline app_dart/lib/src/request_handlers/flaky_handler_utils.dart:427]:
```suggestion
    Team.unknown || null => null,
```

**Review by zanderso — APPROVED** [review]
lgtm w/ nit

File: app_dart/lib/src/request_handlers/flaky_handler_utils.dart (hunk 2 above)
Comment (keyonghan) [reply app_dart/lib/src/request_handlers/flaky_handler_utils.dart]: Done.

**Pull request merged**